text/0003-move_info.md

```diff
@@ -0,0 +1,41 @@
+# Overview
+
+We sometimes want to know the performance of a bot, and *node count*, *nodes created per second (nps)*, *depth*
+are well-known good indexes to know that. This `move_info` feature will add an object to `suggestion` message that contains these
+values.
+
+# Reference
+
+## `suggestion` Message Additions
+
+Attribute   | Type     | Description
+---------   | -------- | ----
+`move_info` | `object` | An object which contains the move info.
+
+A move_info is an object which may contain the following attributes:
+
+Attribute | Type                    | Description
+--------- | ----------------------- | ----
+`nodes`   | `number` or `undefined` | Total number of nodes of current game tree.
+`nps`     | `number` or `undefined` | Number of nodes created per second.
+`depth`   | `number` or `undefined` | The depth of current game tree.
+`extra`   | `string` or `undefined` | Additional informational string about the provided move.
```

Comment:
The "or `undefined`" bit might a bit confusing as `undefined` is not a valid JSON value. I think these should just say `number`/`string` - it's not like the bot has to specify the fields they aren't providing values for anyway.

Reply:
fixed